### PR TITLE
Monkeymode fixes

### LINKED ISSUE #33793: Monkeychat includes .k in the message
title

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
fix: Monkey chat should no longer have .k prefixed
fix: Monkey leaders should actually get jungle fever now
/:cl:

Fixes #33793

### CODE REVIEW

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -55,11 +55,6 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	"å" = "Syndicate",
 	"í" = "CentCom",
 
-	// Species
-	"è" = "binary",
-	"ï" = "changeling",
-	"ô" = "alientalk",
-
```
Comment (coiax) [inline code/modules/mob/living/say.dm]: ???

Comment (MoreRobustThanYou) [reply code/modules/mob/living/say.dm]: isn't need anymore due to saymode datums

Comment (The-clown-known-as-Pepe) [reply code/modules/mob/living/say.dm]: )))

Comment (ninjanomnom) [reply code/modules/mob/living/say.dm]: You're missing the other place this is set some lines above then

Comment (MoreRobustThanYou) [reply code/modules/mob/living/say.dm]: either way it doesn't really matter

Comment (ninjanomnom) [reply code/modules/mob/living/say.dm]: do both or do neither

Comment (MoreRobustThanYou) [reply code/modules/mob/living/say.dm]: k

**Comment by optimumtact** [comment]
Okay I'll stop ignoring you

**Comment by PKPenguin321** [comment]
~~~

**Comment by ninjanomnom** [comment]
I don't feel like checking if those saymode keys are unnecessary now and since this is @MoreRobustThanYou we're talking about here either another maintainer can check and merge it or a testmerge can happen and I'll merge it myself assuming no issues.

**Comment by MoreRobustThanYou** [comment]
@ninjanomnom The saymode keys are contained in the datums themselves now

**Comment by Cyberboss** [comment]
If this breaks say again I'm putting you on PR approval

**Pull request merged**